masonry/src/widget/zstack.rs

```diff
@@ -0,0 +1,324 @@
+// Copyright 2024 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    vello::Scene, widget::WidgetMut, AccessCtx, BoxConstraints, LayoutCtx, PaintCtx, Point,
+    QueryCtx, RegisterCtx, Size, Widget, WidgetId, WidgetPod,
+};
+use accesskit::{NodeBuilder, Role};
+use smallvec::SmallVec;
+use tracing::trace_span;
+
+struct Child {
+    widget: WidgetPod<Box<dyn Widget>>,
+}
+
+#[derive(Default)]
+pub struct ZStack {
+    children: Vec<Child>,
+    alignment: Alignment,
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum Alignment {
+    TopLeading,
+    Top,
+    TopTrailing,
+    Leading,
+    Center,
+    Trailing,
+    BottomLeading,
+    Bottom,
+    BottomTrailing,
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum VerticalAlignment {
+    Top,
+    Center,
```

Comment:
```suggestion
#[derive(Debug, Clone, Copy, PartialEq, Eq)]
pub enum VerticalAlignment {
    Top,
    #[default]
    Center,
```
See: https://doc.rust-lang.org/std/default/trait.Default.html#enums

Our MSRV is higher than 1.62.

Reply:
Thanks for the tip, I've changed it